File: src/main/java/com/github/dkurata38/open_web_library/infra/repository/BookRepositoryImpl.java
package com.github.dkurata38.open_web_library.infra.repository;

import com.github.dkurata38.open_web_library.application.book.BookRepository;
import com.github.dkurata38.open_web_library.domain.book.Book;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public class BookRepositoryImpl implements BookRepository {

	@Override
	public List<Book> findBy() {
		return null;
	}
}

File: repository/src/main/kotlin/com/github/dkurata38/open_web_library/repository/book/BookRepositoryImpl.kt
package com.github.dkurata38.open_web_library.repository.book

import com.github.dkurata38.open_web_library.application.book.BookRepository
import com.github.dkurata38.open_web_library.domain.book.Book
import com.github.dkurata38.open_web_library.domain.book.ISBN
import org.springframework.stereotype.Repository

@Repository
class BookRepositoryImpl : BookRepository {
	override fun findByISBN(isbn: ISBN): Book? {
		TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
	}

	override fun save(book: Book) {
		TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
	}
}
